resources/js/Pages/Dashboard.vue

```diff
@@ -78,12 +90,13 @@ import { ref } from 'vue';
                 posts:[],
                 comments:[],
                 post_content: '',
-                allcomments: []
+                entryPresnt: false,
+                loggedInUser:0,
+                postLength: 3,
             }
         },
         mounted() {
-            this.getAllPosts();
-            this.getAllComments();
+            this.getAllPosts();//fetch all post on page load
```

Comment:
Really what we should be doing here is fetching a specific number of posts rather than every single one - then your Load More functionality should be requesting additional posts from the backend rather than just showing hidden posts.

Reply:
I have advised Tejasvi to ignore this comment for now - we will take this concept into a one-to-one chat.